pkg/services/vmoperator/vmopmachine_test.go

```diff
@@ -513,7 +526,7 @@ var _ = Describe("VirtualMachine tests", func() {
 			Expect(vmopVM.Spec.Volumes).To(HaveLen(2))
 
 			for i, volume := range vsphereMachine.Spec.Volumes {
-				name := volumeName(vsphereMachine, volume)
+				name := fmt.Sprintf("%s-%s", vsphereMachine.Name, volume.Name)
```

Comment:
q why are we dropping the volumeName func?

Reply:
Same as above, reverted.